src/core/event.js

```diff
@@ -673,19 +673,21 @@ Candy.Core.Event = (function(self, Strophe, $) {
 				 * Also triggered when the local client gets kicked or banned from a room.
 				 *
 				 * Parameters:
-				 *   (String) roomJid - Room
-				 *   (String) roomName - Name of room
+				 *   (Candy.Core.Chatroom) room -
 				 *   (String) type - Presence type [kick, ban, leave]
 				 *   (String) reason - When type equals kick|ban, this is the reason the moderator has supplied.
-				 *   (String) actor - When type equals kick|ban, this is the moderator which did the kick
+				 *   (Candy.Core.Chatuser) actor - When type equals kick|ban, this is the moderator which did the kick
 				 *   (Candy.Core.ChatUser) user - user which leaves the room
 				 */
+
+				var room = Candy.Core.getRoom(roomJid)
+				var actorUser = Candy.Core.getRoster(actor);
```

Comment:
If it doesn't exist in our roster, we'd have to do similarly to elsewhere and allocate a temporary object. We should extract this somewhere common.